agent/tool-scripts/datalog/turbostat-datalog

```diff
@@ -1,6 +1,23 @@
 #!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; tab-width: 8 -*-
 
-interval=$1
-file=$2
+PROG="$(basename ${0})"
 
-/usr/bin/turbostat -i $interval | pbench-log-timestamp > $file
+interval="${1}"
+if [[ -z "${interval}" ]]; then
+	printf -- "%s: missing required 'interval' argument\n" "${PROG}" >&2
+	exit 1
+fi
+
+command -v turbostat > /dev/null
+if [[ ${?} -ne 0 ]]; then
+	printf -- "%s: missing required 'turbostat' command\n" "${PROG}" >&2
+	exit 1
+fi
+
+if grep -q KVM /sys/devices/virtual/dmi/id/product_name; then
```

Comment:
systemd-virt-detect

Reply:
Seems to be everywhere `systemd` is.  I'll add a patch that shows just that change to use `systemd-detect-virt`.